forte/ontology_specs/base_ontology.json

```diff
@@ -467,6 +467,58 @@
       "parent_entry": "forte.data.ontology.top.Payload",
       "description": "A payload that caches image data",
       "attributes":[]
+    },
+    {
```

Comment:
Let's create another spec file for these metadata, like `payloads`. (also format the json a little).